scene/3d/sprite_3d.cpp

```diff
@@ -716,8 +745,8 @@ Sprite3D::Sprite3D() {
 
 void AnimatedSprite3D::_draw() {
 
-	RID immediate = get_immediate();
-	VS::get_singleton()->immediate_clear(immediate);
+	RID mesh = get_mesh();
+	VS::get_singleton()->mesh_clear(mesh);
```

Comment:
Definitely would not clear the mesh, instead would update the surface.

Reply:
Done!